### PR TITLE
[14.0][MIG] account_asset_non_deductible

### PR DESCRIPTION
**Pull request by ChrisOForgeFlow**
Standard Migration

Depends on:

- #55 
- #61

### CODE REVIEW

File: account_asset_non_deductible/models/account_move.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        return vals
+
+
+class AccountMoveLine(models.Model):
+    _inherit = "account.move.line"
+
+    def _get_asset_base_value(self):
+        self.ensure_one()
+        value = self.balance
+        tax_non_deductible_percent = self.tax_ids.get_non_deductible_percent(
```
Comment (mariadforgeflow) [inline account_asset_non_deductible/models/account_move.py:27]: `get_non_deductible_percent` doesn't exist anymore in https://github.com/nuobit/odoo-addons/pull/154.

Comment (MiquelRForgeFlow) [reply account_asset_non_deductible/models/account_move.py:27]: :(

Comment (eantones) [reply account_asset_non_deductible/models/account_move.py:27]: I restored it

File: account_asset_non_deductible/models/account_move.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            self.move_id.date or self.move_id.invoice_date,
+            self.move_id.company_id.id or self.env.company.id,
+        )
```
Comment (eantones) [inline account_asset_non_deductible/models/account_move.py]:
```suggestion
        tax_non_deductible_percent = self.tax_ids.get_non_deductible_percent(
            self.move_id.date or self.move_id.invoice_date,
            self.move_id.company_id.id or self.env.company.id,
            self.move_id.move_type in ("out_refund", "in_refund"),
        )
```

**Pull request merged**